apps/admin/frontend/src/components/export_election_ballot_package_modal_button.test.tsx

```diff
@@ -94,8 +112,15 @@ test('Button renders properly when not clicked', () => {
 
 test('Modal renders insert usb screen appropriately', async () => {
   const usbStatuses: UsbDriveStatus[] = ['absent', 'ejected', 'bad_format'];
+  // Need 1 ApiMock per test, so we recreate the global-scoped mock to clobber
+  // the expectations we created in `beforeAll()`.
+  // TODO add an explicit 'clear expectations' function
+  apiMock = createApiMock();
```

Comment:
Curious why we need this when we I _think_ we could just do

```
apiMock = createApiMock();
```

(without the `const`) inside the `usbStatus` loop below

Reply:
Got it!

One of the pros to the current pattern (of the global mock variable but a fresh mock for each test):

```ts
let apiMock: ApiMock;

beforeEach(() => {
  apiMock = createApiMock();
});

afterEach(() => {
  apiMock.assertComplete();
});
```

is that we guarantee that `apiMock.assertComplete()` is called - something that's easy to forget when you're doing, say:

```ts
test(() => {
  const apiMock = createApiMock();
  ...
  // Have to remember to call this
  apiMock.assertComplete();
});
```

-----

Taking a closer look at this current case, a better solution that eliminates the need for any overrides is to just run each of these USB status tests as a separate test using a `test.each` (here's an [example](https://github.com/votingworks/vxsuite/blob/87da7bf630353891a84a81c39ef5ed8a9af7c538/libs/auth/src/dipped_smart_card_auth.test.ts#L231-L260))